rustler_codegen/src/ex_struct.rs

```diff
@@ -5,7 +5,7 @@ use syn::{self, spanned::Spanned, Field, Ident};
 use super::context::Context;
 use super::RustlerAttr;
 
-pub fn transcoder_decorator(ast: &syn::DeriveInput) -> TokenStream {
+pub fn transcoder_decorator(ast: &syn::DeriveInput, add_exception: bool) -> TokenStream {
```

Comment:
I am not a huge fan of a boolean `add_exception` flag for this, but aside from copying the module as a whole, I can't think of a cleaner way. @rusterlium/core maybe someone else has an idea to avoid this flag?